src/index.js

```diff
@@ -396,18 +428,27 @@ function getTranslationsNOIMaps() {
 			searchElementsStarterNOIMaps();
 			translateElementsNOIMaps();
 		} else {
-			//location.reload();
-			console.log("400 error: reload ");
+			if(debugActive == 1) {
+				console.log('Error on getTranslationsNOIMaps fetching translations');
+			} else {
+				location.reload();
```

Comment:
This could lead to an endless reloading of the web component... are you sure you want to do this?